### DISCUSSION TITLE
Alexa建議小女孩做危險挑戰

### DISCUSSION THREAD

**Discussion by ChinChun0220** [1101-GS4719-C/Python-Community-GS4719-C#231] [科技新知]
人工智能、智慧管家，似乎漸漸進入我們生活，也帶來許多方便。但智慧管家畢竟還不是人，無法判斷收到的數據是否危險，一不小心就有可能造成嚴重傷害。

https://www.bbc.com/news/technology-59810383?fbclid=IwAR3ZfgBRT63NZRUpoH9_274lNnQLH3u3HCJppFUqMLcPKToEMFBgi0OJ6n8

BBC的一篇報導提到，小女孩問Alexa:[有甚麼挑戰可以做?]
Alexa回復 "Plug in a phone charger about halfway into a wall outlet, then touch a penny to the exposed prongs" .
源自於抖音之類社群軟體的危險挑戰。

幸好小女孩並沒有跟著做才沒有導致真的傷害，但也讓人沈思，這類智慧管家只能傳遞"found on the web"的資訊，無法判斷正確或危險性，雖然亞馬遜公司馬上就修復這個問題，並保證之後有類似的錯誤會馬上修正。但目前還是無法馬上根治。

我就在思考一個問題，若未來，能夠有個人工智慧可以判斷每件事情的危險性及正確後篩選網路上的資訊，這樣一定好嗎?

**Comment by HSU-YU-JEN** [comment]
想問問看，Alexa會做這類的錯誤指示是，因為在網路演算法上的錯誤學習嗎

**Reply by ChinChun0220** [reply]
看文章中應該是指Alexa幫忙搜尋「挑戰」然後給出第一個答案，但那種危險挑戰的確不應該被演算法推送到第一個。

**Comment by JAIMEHUNG** [comment]
如果只能在網路上抓資訊，會產生讓未成年者接觸到敏感性資訊的問題，他們沒有判別能力而容易受到影響。

**Reply by ChinChun0220** [reply]
這時候就要思考是否要限制網頁上的資訊要有年齡限制，例如fb有時候會擋敏感 血腥圖片，但尺度要到哪就很難有個定數

**Comment by luo910227** [comment]
這真的是個值得思考的問題，我認為因為人工智慧能夠可以判斷每件事情的危險性及正確後篩選網路上的資訊之前，仍然必須經過學習，而如果學習的過程中出了甚麼差錯，真的開始使用時，如果使用者本身對於人工智慧的建議不都有獨立思考的能力判斷，可能造成極大的問題，因此我覺得有這種功能的人工智慧就如同複製人，有進步性不過仍存在爭議及危險性，不太適合發展。

**Reply by ChinChun0220** [reply]
這也會造成一個問題，若人們在訓練人工智慧的時候就帶有某種偏見（或錯誤），有可能造成可怕的後果，倫理上爭議很多

**Comment by Isongzhe** [comment]
我認為這樣會不會造成，所謂的危險性跟正確的事情 ，變成只有AI自己所認為的才是正確的。

**Reply by ChinChun0220** [reply]
我認為很有可能… 這就是為什麼AI可怕的原因…

**Comment by cody910120** [comment]
喔穴那這樣真的米奇不妙屋

**Comment by YuHengChen54** [comment]
感覺很多人工智慧的系統都有這個問題

**Comment by vvvioletta** [comment]
我覺得可怕的是AI一旦被餵到錯誤的資訊 有可能就一直深化下去 最後變成亂源（就跟從小沒教好小孩 長大就改不太回來了 只是AI還有機會格式化）

**Comment by BryanNee** [comment]
Alexa 真的真的算是現代科技的雙面刃

**Comment by pekoranoinu** [comment]
AI如果可以判斷每件事情的危險性及正確的話，那我們人類就不需要去思考了，這樣對人類會是個大問題。

**Comment by chupochun** [comment]
我認為人類還不能太過相信AI 畢竟還沒有訂立一個完善的使用規範 大家都應該保有懷疑的態度
